src/main/groovy/com/ofg/uptodate/finder/jcenter/JCenterNewVersionFinderFactory.groovy

```diff
@@ -50,6 +51,11 @@ class JCenterNewVersionFinderFactory implements NewVersionFinderFactory {
         } as Future
     }
 
+    //application/unknown is returned from jCenter when using CloudFront - #42
+    private void handleUnknownContentTypeAsXml(HTTPBuilder httpBuilder) {
```

Comment:
Content type is not `unknown` or not known but `application/unknown`, let's reflect that in the method name